Drivers/Src/LTC6811.c

```diff
@@ -76,20 +77,20 @@ Copyright 2017 Linear Technology Corp. (LTC)
 /*********************************************************/
 /*** Code that was added by UTSVT. ***/
 /*********************************************************/
-static OS_MUTEX MinionsASIC_Mutex;
+OS_MUTEX MinionsASIC_Mutex;
 
 void LTC6811_Init(cell_asic *battMod){
+  //only create the mutex the first time this function is called (called by Voltage_Init() and Temperature_Init())
```

Comment:
We only have one thread for the two right?

Reply:
Yes, that is correct.